src/styles/cards.scss

```diff
@@ -0,0 +1,133 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  padding: 32px 16px 16px;
+  border: 1px solid $card-bd-color;
+  border-radius: 5px;
+  transition: all 300ms ease;
```

Comment:
list all styles to which you apply transition instead of using 'all', code style - transition